php/files/apcu.ini

```diff
@@ -1,3 +0,0 @@
-extension=apcu.so
```

Comment:
I don't see this extension in the new ini files. Are you sure you can remove it?

Reply:
It is already present by default, I was overriding it.

Now it is still there, but I just activated apc for CLI, without touching the original conf.